### ISSUE TITLE
.NET Framework packages now take dependency on Microsoft.NETFramework.ReferenceAssemblies

### ISSUE THREAD

**Issue by dotMorten** [novotnyllc/MSBuildSdkExtras#254]
NuGet packages generated with 3.0.22 will cause .NET Framework targets to take a dependency on `Microsoft.NETFramework.ReferenceAssemblies` in the nuspec.

It seems to me `ExtrasImplicitPlatformPackageIsPrivate` was meant to default to `true` and not `false`.

See discussion:
https://github.com/novotnyllc/MSBuildSdkExtras/commit/77e3f3b9b2484b250ff734244732a699f1351411#commitcomment-45713712

Workaround:
```xml
 <PropertyGroup Condition="'$(TargetFramework)'=='net461'">
    <ExtrasImplicitPlatformPackageIsPrivate>true</ExtrasImplicitPlatformPackageIsPrivate> 
 </PropertyGroup>
```

**Comment by clairernovotny** [comment]
Fixed in 3.0.23

**Comment by dotMorten** [comment]
You rock! ❤️
